api/v1alpha1/openapi.yaml

```diff
@@ -4999,7 +4999,7 @@ components:
           format: byte
           type: string
         signerName:
-          description: Indicates the requested signer, and is a qualified name.
+          description: Indicates the requested signer, and is a qualified name. Supported values are "enrollment" and "renewal".
```

Comment:
We don't have a backend implementation for `renewal` signer yet, so I would remove this option until the implementation is added again.